test/wasm/assets/015_results.yaml

```diff
@@ -16,18 +16,14 @@ cases:
   query: input[i] > 0
   input: [0,1,1]
   want_result: [{'i': 1}, {'i': 2}]
-- note: wildcard vars excluded
-  query: a = input[_]; a > 1
-  input: [1,2,3]
-  want_result: [{'a': 2}, {'a': 3}]
 - note: locally scoped vars rewritten (assignment operator)
   query: a := input[_]; a > 1
   input: [1,2,3]
-  want_result: [{'a': 2}, {'a': 3}]
+  want_result: [{'$0':2, 'a':3},{'$0':1, 'a':2}]
 - note: locally scoped vars rewritten (some keyword)
   query: some a; a = input[_]; a > 1
   input: [1,2,3]
-  want_result: [{'a': 2}, {'a': 3}]
+  want_result: [{'$0':2, 'a':3},{'$0':1, 'a':2}]
```

Comment:
Hmm! I was wondering if this might be something we'd want to do something about, but I don't mind the generated vars at all, I think. 💭

Reply:
I'm guessing this is due to the changes in the planner. I don't think we want values for wildcard variables to be included in the result. If the query contains multiple wildcards, the user won't know what values correspond to what wildcards.